wgsl/index.bs

```diff
@@ -316,19 +314,75 @@ However, UTF-8 is always a valid encoding for a [SHORTNAME] program.
 Note: The intent of promoting UTF-8 like this is to simplify interchange of [SHORTNAME] programs
 and to encourage interoperability among tools.
 
+[SHORTNAME] program text consists of a sequence of characters, grouped into contiguous non-empty sets forming:
+* [=comments=]
+* [=tokens=]
+* [=blankspace=]
+
+<dfn>Blankspace</dfn> is any combination of one or more of the following characters:
+* space
+* tab
+* linefeed
+* vertical tab
+* formfeed
+* carriage return
+
+Issue: What else should be blankspace? Anything in the Unicode Separator category?
+
+To parse a [SHORTNAME] program:
+1. Remove comments:
+    * Replace the first comment with a space character.
+    * Repeat until no comments remain.
+2. Scanning from beginning to end, group the remaining characters into tokens and blankspace
+    in greedy fashion:
+    * The next group is formed from the longest
+        non-empty prefix of the remaining ungrouped characters, that is either:
+        * a valid token, or
+        * blankspace
+    * Repeat until no ungrouped characters remain.
+3. Discard the blankspace, leaving only tokens.
+3. Parse the token sequence, attempting to match the `translation_unit` grammar rule.
+
+<pre class='def'>
+translation_unit
+  : global_decl_or_directive*
```

Comment:
Nice one, better without `EOF`!

Reply:
Yeah, thanks for noticing.  :-)

It's replaced by the rule that tokenization and parsing must process and match the entire source text.